src/main/java/com/gachigang/ontherun/common/ApplicationConstants.java

```diff
@@ -28,5 +28,7 @@ public class Validation {
     public class Security {
         public static final String TOKEN_PREFIX = "Bearer ";
         public static final String AUTH = "auth";
+        public static final String[] ALLOWEDORIGINS = {"https://ontherun.com"};
```

Comment:
These constants should be in application.properties file (Because we may want to change it without rebooting our application), thus it is better to move in this file

Reply:
Ok, i will replace it